Source/WebCore/Modules/url-pattern/URLPatternParser.cpp

```diff
@@ -330,5 +329,219 @@ String escapeRegexString(StringView input)
     return escapeRegexStringForCharacters(input.span16());
 }
 
+// https://urlpattern.spec.whatwg.org/#convert-a-modifier-to-a-string
+String convertModifierToString(Modifier modifier)
+{
+    switch (modifier) {
+    case Modifier::ZeroOrMore:
+        return "*"_s;
+    case Modifier::Optional:
+        return "?"_s;
+    case Modifier::OneOrMore:
+        return "+"_s;
+    default:
```

Comment:
Do we need the default case here?

Reply:
Yes because this doesn't account for all the values, eg Modifier::None. In same cases, we want to return nothing.